.github/workflows/ci_python.yml

```diff
@@ -75,6 +79,10 @@ jobs:
           poetry run pydocstyle pysrc
 
   debug:
+    if: |
```

Comment:
Only debug on pull requests, merge queue, and pushes to main (since it is needed for docs).